native/src/components/PoiDetails.tsx

```diff
@@ -70,12 +70,12 @@ const PoiDetails = ({ poi, feature, language }: PoiDetailsProps): ReactElement =
 
   const openExternalMaps = () => {
     const externalMapsUrl = getExternalMapsLink(poi.location, Platform.OS)
-    openExternalUrl(externalMapsUrl).catch(() => showSnackbar(t('error:noSuitableAppInstalled')))
+    openExternalUrl(externalMapsUrl).catch(() => showSnackbar({ text: t('error:noSuitableAppInstalled') }))
   }
 
   const copyLocationToClipboard = (): void => {
     Clipboard.setString(`${address}, ${postcode} ${town}`)
-    showSnackbar(t('addressCopied'))
+    showSnackbar({ text: t('addressCopied') })
```

Comment:
why do we need the `SnackbarType` that includes `text:string` instead of just passing a `string`

Reply:
Makes it more flexible, in the future we quite sure will need to pass additional info to the snackbar (buttons and onClick methods for example). E.g. for this issue: https://issues.tuerantuer.org/browse/IGAPP-328

Since I was already refactoring I figured I could already change the types/usages here.